daphne/server.py

```diff
@@ -199,7 +197,7 @@ def create_application(self, protocol, scope):
         assert "application_instance" not in self.connections[protocol]
         # Make an instance of the application
         input_queue = asyncio.Queue()
-        application_instance = self.application(scope=scope, lock=self.lock)
+        application_instance = self.application(scope=scope)
```

Comment:
This is unused right? (Created on line 205)

Not expecting any changes in this file...

Reply:
Just pushed another commit a half hour or so ago; maybe you need to refresh. There was no reason for me to have changes in `daphne/server.py` that is correct, I had just not noticed I was in that file as I was threading the lock through as an argument. In latest commit the changes are confined to `daphne/testing.py`